app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      include Wor::Paginate
+      before_action :authenticate_user!
+
+      def index
+        render_paginated Book.all, each_serializer: BooksListSerializer
```

Comment:
You should not specify the serializer

Reply:
According to the Apiari documentation, the response of both endpoints (index and show), return different forms of the Book object hence the need of specifying the serializer.

Reply:
The Apiari documentation has been deprecated in concern to this new training we are doing 

![image](https://user-images.githubusercontent.com/11698687/53956640-eca76980-40ba-11e9-8a76-d0eb1e86ae4c.png)

Use the same serializer to show or for index in this case an as we go on in the training we could decide which is the best approach to the diferents controllers